optuna/samplers/_gp/sampler.py

```diff
@@ -214,42 +229,80 @@ def sample_relative(
             normalized_params,
         ) = gp_search_space.get_search_space_and_normalized_params(trials, search_space)
 
-        _sign = -1.0 if study.direction == StudyDirection.MINIMIZE else 1.0
-        score_vals = np.array([_sign * cast(float, trial.value) for trial in trials])
-
-        score_vals = gp.warn_and_convert_inf(score_vals)
-        standardized_score_vals = (score_vals - np.mean(score_vals)) / max(EPS, np.std(score_vals))
+        _sign = np.array([-1.0 if d == StudyDirection.MINIMIZE else 1.0 for d in study.directions])
+        standardized_score_vals, _, _ = _standardize_values(
+            values=gp.warn_and_convert_inf(_sign * np.array([trial.values for trial in trials]))
```

Comment:
Since `gp.warn_and_convert_inf` is called at the beginning of `_standardize_values`, this call is duplicated.
```suggestion
            _sign * np.array([trial.values for trial in trials])
```